Kentico.Kontent.Delivery.Abstractions/ContentLinks/ContentLinkResolver.cs

```diff
@@ -3,7 +3,7 @@
 using System.Text.RegularExpressions;
 using Newtonsoft.Json.Linq;
 
-namespace Kentico.Kontent.Delivery.ContentLinks
+namespace Kentico.Kontent.Delivery.Abstractions.ContentLinks
 {
     internal sealed class ContentLinkResolver
```

Comment:
I don't believe this class belongs to the abstractions...it contains functionality. we should either move it back or "interfacialize" it...is that a word?

Reply:
@petrsvihlik  @alesk-kentico I just moved a ContentLinkResolver to the Delivery library and change how to get IContentLinkUrlResolver instance - IContentLinkResolver contains property IContentLinkUrlResolver.